code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -111,7 +113,48 @@ Difficulty: Hard
 	chosen_message = "<span class='colossus'>You are now warping to blood around your clicked position.</span>"
 	chosen_attack_num = 4
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/OpenFire()
+/mob/living/simple_animal/hostile/megafauna/bubblegum/enrage()
+	. = ..()
+	maxHealth = 2000 //Less health, as a phase 2
+	health = 2000
+	rapid_melee = 12 //Don't stand still
```

Comment:
Are these deciseconds? Not sure on this one

Reply:
Good question. Let me find out